### ISSUE TITLE
[FEATURE] - Split out flightlog into `time` & `event`

### ISSUE THREAD

**Issue by jordanhavard** [invernyx/smartcars-3-bugs#63]
**Is your feature request related to a problem? Please describe.**
Looking at the flight log when submitting a PIREP, its being sent as an array of elements with each element being a new log line and this can make it hard to parse the actual event against the time.

**Describe the solution you'd like**
Something like the below for each log entry
```
[
	'time' => '2023-03-25T12:42:44.392Z',
	'event => 'Flying PMDG 737-800 Qantas Airways (VH-VXA | 2016)',
]
```
instead of 
```
 '2023-03-25T12:42:44.392Z - Flying PMDG 737-800 Qantas Airways (VH-VXA | 2016)' all on the one line
```

**Describe alternatives you've considered**
splitting out the string by ` - ` however this could cause issues if that is matched anywhere in the event text

**Additional context**
smartCARS2 wrapped the time in `[]` however this again caused issued if that character was present in the aircraft name

**Comment by turbofandude** [comment]
Although I see the advantage of this, given the scope of this change (breaking all existing installs) I think the best bet is to split the string at '-' and **take only the first result** (i.e. limit the results to 2). Due to this, this will be a "won't fix" because there is a viable alternative. Thank you for the feedback!
